instance/config.py

```diff
@@ -0,0 +1,38 @@
+import os
+
+
+class Config(object):
+    """Parent configuration class."""
+    DEBUG = False
+    CSRF_ENABLED = True
+    SECRET = os.getenv('SECRET')
+
+
+class DevelopmentConfig(Config):
+    """Configurations for Development."""
+    DEBUG = True
+
+
+class TestingConfig(Config):
+    """Configurations for Testing, with a separate test database."""
+    TESTING = True
+    DEBUG = True
+
+
+class StagingConfig(Config):
```

Comment:
What's the difference between the `development` configuration and the `staging` configuration?